source/documentation/airflow.md

```diff
@@ -207,6 +251,9 @@ dag = DAG(
     schedule_interval=None,
 )
 
+# It is good practice to use the same name for teh task_id and task_name.
```

Comment:
Typo: teh

Reply:
Also:  `task_name` or `name` ?

Reply:
Have fixed and clarified

Reply:
OK, should line 267 be:
`name=task_id`
or
`task_name=task_id`